backend/src/Edna/Config/Environment.hs

```diff
@@ -0,0 +1,104 @@
+module Edna.Config.Environment
+  ( EdnaEnvironments(..)
+
+  , ednaEnvs
+  , askDebugDB
+  ) where
+
+import Universum
+
+import Control.Exception (throwIO)
+import Data.ByteString.Char8 (pack)
+import qualified Data.Char as C
+import Edna.Config.Definition (LoggingConfig(..))
+import Edna.Util (ConnString(..), NetworkAddress)
+import System.Environment (lookupEnv)
+import qualified Text.Show
+
+data EnvParseError = WrongNetworkAddress
+                   | WrongMaxConnections
+
+instance Show EnvParseError where
+  show WrongNetworkAddress = "Can't parse EDNA_API_LISTEN"
+  show WrongMaxConnections = "Can't parse EDNA_DB_MAX_CONNECTIONS"
+
+instance Exception EnvParseError
+
+readEnv :: Read a => String -> EnvParseError -> IO (Maybe a)
+readEnv name err = do
+  env <- lookupEnv name
+  case readEither <$> env of
+    Nothing -> return Nothing
+    Just (Right value) -> return $ Just value
+    Just (Left _) -> throwIO err
+
+getBoolEnv :: String -> IO (Maybe Bool)
+getBoolEnv name = lookupEnv name <&> \case
+  Nothing                        -> Nothing
+  Just "1"                       -> Just True
+  Just (map C.toLower -> "true") -> Just True
+  _                              -> Just False
+
+-- If @EDNA_DB_DEBUG@ environment variable is set to @1@
+-- or @TRUE@ (case-insensitive), we will run postgres with
+-- debug logging.
+askDebugDB :: IO Bool
+askDebugDB = do
+  env <- getBoolEnv "EDNA_DB_DEBUG"
+  return $ fromMaybe False env
+
+data EdnaEnvironments = EdnaEnvironments
```

Comment:
Let me repeat myself:
> Why are these conversations marked as resolved if they are not? I mean, there are no requested changes (maybe you forgot to push?). And usually the person who requested changes marks a conversation as resolved after checking that changes have been made properly.

Reply:
No, I do not forgot to push, just wanted to make some more changes in commit.
I mark it as resolved because I agree with you and made changes locally. So it's just easy to read other comments in PR.
If it is a problem, I will not close it)

Reply:
The problem with resolving before the reviewer checked the change is that it makes it harder for review to re-review PR after some changes have been made. Normally I (or another reviewer) write some comments, you make changes, I come to the PR again and go through my comments checking which of them have been fixed in some way (by making changes in code or explaining why no changes are needed or by some other means). If my comments are marked as resolved, I need to unfold each of them to recall what I was requesting.
It's ok to resolve comments where no changes are requested, e. g. sometimes I can write "I like how it's done" and you can resolve it as soon as you read it.

> So it's just easy to read other comments in PR.

For this purpose I suggest that you just write "Done" or "Ok" for comments that you already took into account. Then you can go through all comments and pay attention only to those not ending with "Done" or "Ok" from your side.

Reply:
Ok)